x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -84,3 +84,5 @@ export const ERROR_CONNECTING_HEADER = 'x-ent-search-error-connecting';
 export const READ_ONLY_MODE_HEADER = 'x-ent-search-read-only-mode';
 
 export const ENTERPRISE_SEARCH_KIBANA_COOKIE = '_enterprise_search';
+
+export const LOGS_SOURCE_ID = 'ent-search-logs';
```

Comment:
I copied this pattern from the monitoring plugin ([1](https://github.com/elastic/kibana/blob/ca17d931deaff7217dc7f75ae1ad6ff3be680543/x-pack/plugins/monitoring/common/constants.ts#L139-L142) / [2](https://github.com/elastic/kibana/blob/ca17d931deaff7217dc7f75ae1ad6ff3be680543/x-pack/plugins/monitoring/server/lib/logs/init_infra_source.ts)) - I'm not totally sure if there's a specific casing (camel? snake?) we should be using, or why they chose kebab 😬